### ISSUE TITLE
Fix false positives for WebExtension replacement keywords in value-keyword-case

### ISSUE THREAD

**Issue by fregante** [stylelint/stylelint#4707]
> Clearly describe the bug

Chrome/Firefox extensions have CSS keywords that are replaced at runtime and are case-sensitive. For example:

```css
body {
	direction: __MSG_@@bidi_dir__;
}
```

Which the browser interprets as:

```css
body {
	direction: ltr;
	/* or direction: rtl; */
}
```

These are mostly used for localization/internalization. Documentation: https://developer.chrome.com/extensions/i18n

> Which rule, if any, is the bug related to?

`value-keyword-case`

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "value-keyword-case": "lower"
  }
}
```

> Which version of stylelint are you using?

`13.3.2`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Kinda. However it's parsed by the browsers, not a transpiler.

> What did you expect to happen?

Keywords that match `/__MSG_[^\s]+__/` should be skipped by the rule.

> What actually happened (e.g. what warnings or errors did you get)?

```shell
test.css
 2:13  ✖  Expected "__MSG_@@bidi_dir__" to be "__msg_@@bidi_dir__"   value-keyword-case
```

**Comment by fregante** [comment]
NB: the lower case version is not compatible with browsers.

**Comment by jeddy3** [comment]
@fregante Thanks for the clear report and for using the template.

I can reproduce this [using the demo](https://stylelint.io/demo#N4Igxg9gJgpiBcIBG0CeACYAdAdu9UAlgE4xgAuhEO86A+nQLIDKA4nQAIdKFF1HEGAblwBfEABoQAZ1Q5yAQwAeCcNOmTw1AGaEA5qux50WEMQCuAGxjTTtYPlz5HIAG4LL5mAFoA1jFQAdwhiKG8wBWkYOxMQSwhAmGJTJ3RRMRBRIA).

> Keywords that match `/__MSG_[^\s]+__/` should be skipped by the rule.

Sounds good. It can be added to the [`isStandardSyntaxValue` util](https://github.com/stylelint/stylelint/blob/master/lib/utils/isStandardSyntaxValue.js).

I've labelled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.

**Comment by malsf21** [comment]
Is anybody else working on this? If not, I can give it a shot!

**Comment by jeddy3** [comment]
> I can give it a shot!

Excellent. I'll label as "wip" for you.
